src/utils/time.ts

```diff
@@ -324,3 +324,17 @@ export const convertUtcSecondsFromInputDateAndTime: (input: {
   date: string;
   time: string;
 }) => number = ({ date, time }) => getUnixTime(new Date(`${date} ${time}`));
+
+export const formatDayLabel = (
+  day: Date | number,
+  isScheduleTimeshifted: boolean
+) => {
+  if (isScheduleTimeshifted) {
+    return format(day, "do");
+  }
+
+  return formatDateRelativeToNow(day, {
+    formatOtherDate: (dateOrTimestamp) => format(dateOrTimestamp, "do"),
+    formatTomorrow: (dateOrTimestamp) => format(dateOrTimestamp, "do"),
```

Comment:
These `do` could be made constants in `dateSettings.ts` with proper library prefix `DAYJS_` or `DATEFNS_`.

Kind of important to have longer expressive names for those constants as well - plenty of different formats and places to use them.

Reply:
[0dc4703](https://github.com/sparkletown/sparkle/pull/2813/commits/0dc470385bb9fc6a84f4a1e50a08c986d4edd8a7)